src/eko/msbar_masses.py

```diff
@@ -339,26 +332,28 @@ def evolve(
 
 
 def compute(theory_card):
-    r"""
-    Compute the |MSbar| masses solving the equation :math:`m_{\bar{MS}}(\mu) = \mu`
-    for each heavy quark and consistent boundary contitions.
+    r"""Compute the |MSbar| masses.
+
+    Computation is performed solving the equation :math:`m_{\bar{MS}}(\mu) =
+    \mu` for each heavy quark and consistent boundary contitions.
 
     Parameters
     ----------
-        theory_card: dict
-            theory run card
+    theory_card : dict
+        theory run card
 
     Returns
     -------
-        masses: list
-            list of |MSbar| masses squared
+    masses : list
+        list of |MSbar| masses squared
+
     """
     # TODO: sketch in the docs how the MSbar computation works with a figure.
     nfa_ref = theory_card["nfref"]
 
     q2_ref = np.power(theory_card["Qref"], 2)
     masses = np.concatenate((np.zeros(nfa_ref - 3), np.full(6 - nfa_ref, np.inf)))
-    fact_to_ren = theory_card["fact_to_ren_scale_ratio"] ** 2
+    xif = theory_card["fact_to_ren_scale_ratio"] ** 2
```

Comment:
again: you can just call it `xif2`

Reply:
Fixed in 6dbebad4fc7d94a6fa67df91aa7e2ffaebc8c58e